frontend/src/console/components/PreviewComponent/styles.scss

```diff
@@ -1,7 +1,9 @@
+@import "~bootstrap/scss/functions";
+@import "~bootstrap/scss/variables";
+@import "~bootstrap/scss/mixins";
+@import "~bootstrap/scss/utilities";
```

Comment:
Why we need this, @arjunsinghy96?

Reply:
Thanks for noticing this 🙂 I wanted to make sure that bootstrap utility classes are available for use. I didn't realise that all boostrap css is imported at global app level.
Removing this in update.